src/Registration/Registration.css

```diff
@@ -1,17 +1,10 @@
-.container {
-    width: 100%;
-    height: 100%;
-}
-
-.card {
-    padding: 3em;
-    position: fixed;
-    max-width: 40em;
-    min-width: 30em;
+#registration-card {
+    padding: 2.5rem;
```

Comment:
We should discuss what kind of units we're going to use. Some files use (r)em, px and/or %.

Reply:
Agreed, we could add that as an item to do upcoming sprint.